dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ORCIDHelper.java

```diff
@@ -70,6 +71,8 @@ public final class ORCIDHelper {
     private static final ObjectMapper MAPPER = Jackson.newObjectMapper();
 
     private static String baseApiUrl; // baseApiUrl should result in something like "https://api.sandbox.orcid.org/v3.0/" or "https://api.orcid.org/v3.0/"
+    private static String basePublicUrl; // basePublicUrl should result in something like "https://pub.sandbox.orcid.org/v3.0/" or "https://pub.orcid.org/v3.0/"
```

Comment:
Maybe remove the `https://pub.sandbox.orcid.org/v3.0` example if we always plan to hit the public API